src/components/PostComments.jsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import { fetchUsers, postComments } from "../api";
+import { useParams } from "react-router-dom";
+
+export default function PostComments() {
+  const [body, setBody] = useState("");
+  const [name, setName] = useState("tickle122");
+  const [newComment, setNewComment] = useState([]);
+  const [allUsers, setUsers] = useState([]);
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetchUsers().then(({ users }) => {
+      console.log(users);
+      return setUsers(users);
+    });
+  }, []);
+
+  const handleBody = (event) => {
+    setBody(event.target.value);
+  };
+
+  const handleName = (event) => {
+    setName(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComments(name, body, id).then(({ comment }) => {
+      setBody("");
+      setName("tickle122");
+      return setNewComment([comment]);
```

Comment:
why are you setting newComment to an array here?

It's only ever going to be one comment?

Reply:
while rendering it is showing the titles and CSS before posting the comment so have done that so I can use map and make it appear when I click submit